src/components/WalkThrough.js

```diff
@@ -120,6 +120,14 @@ class WalkThrough extends PureComponent {
                             {/* text="Create continuous predictions for a stream of events"*/}
                             {/* threeLines*/}
                         {/* />*/}
+                         <GuideItem
+                            avatar={<Avatar suffix="pink">8</Avatar>}
+                            button={<Button flat primary href={'/#/explanation'}
+                                            iconEl={<FontIcon>work</FontIcon>}>Explanation</Button>}
+                             title="Explanation"
+                             text="Explanation"
+                             threeLines
+                        />
```

Comment:
Please comment this so that when it is merged and deployed it is available only to those that know the exact url.